intyg-proxy-service/integration-hsa/src/main/java/se/inera/intyg/intygproxyservice/integration/organization/client/HsaOrganizationClient.java

```diff
@@ -71,6 +89,13 @@ public HealthCareUnitMembers getHealthCareUnitMembers(
     return getHealthCareUnitMembersResponseTypeConverter.convert(type);
   }
 
+  private static GetUnitType getUnitParameters(String hsaId) {
+    final var parameters = new GetUnitType();
+    parameters.setUnitHsaId(hsaId);
+    parameters.setIncludeFeignedObject(false);
```

Comment:
We also need to set a Profile right? 
These are our alternatives:
```java
    @XmlEnumValue("basic")
    BASIC("basic"),
    @XmlEnumValue("extended")
    EXTENDED("extended"),
    @XmlEnumValue("all")
    ALL("all");
```
I suggest we set it to ```basic```

Reply:
I tried adding profile before but from what I can see it doesn't have a setter